components/Footer/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+import styles from './Footer.module.scss'
+import { useEffect } from 'react'
+import AOS from 'aos'
+import 'aos/dist/aos.css'
+
+export default function Footer() {
+	useEffect(() => {
+		AOS.init({
+			easing: 'ease-out-quad',
+			duration: 1000,
+		})
+	}, [])
+
+	return (
+		<>
+			<div className={styles.footerContainer}>
+				<div className={styles.newsSection}>
+					<p className={styles.textNews}>najnowsze aktualności</p>
+					<div className={styles.containerNews}>
+						<div data-aos='flip-down' className={styles.news}>
+							<p className={styles.newsShadow}>Pierwszy artykuł</p>
+							<img className={styles.newsImg} src='/images/newsNew.png' alt='first article' />
```

Comment:
I think that we should use in next `<Image/>` components which are delivered by next natively. Update this in all files which you updated.